src/zenml/cli/secret.py

```diff
@@ -206,17 +207,22 @@ def register_secret(
             for k in secret_keys:
                 v = getpass.getpass(f"Secret value for {k}:")
                 if v:
-                    secret_contents[k] = v
+                    secret_contents[k] = expand_argument_value_from_file(
```

Comment:
When adding the ability to expand secret values from file contents, I was thinking of also doing it in the interactive mode, but then decided not to, because 1. you can't see what you're typing and 2. you don't get auto-completion to help you to point to an existing file on disk. These reasons make it really error prone to specify a path in interactive mode.

Reply:
Do you think it's better to remove it again? I've just without checking expected the interactive version to work the same way and tried copy-pasting the paths there with a prepended @ and got confused when it didn't work, that's why I implemented it.